src/component/genericmetadata/CreatePropertyForm.tsx

```diff
@@ -31,54 +33,120 @@ function isValid(data: { iri: string }) {
 const CreatePropertyForm: React.FC<CreatePropertyFormProps> = ({
   onOptionCreate,
   toggleModal,
+  languages,
+  language,
 }) => {
   const { i18n } = useI18n();
   const [iri, setIri] = useState("");
-  const [label, setLabel] = useState("");
-  const [comment, setComment] = useState("");
-  const language = useSelector(
-    (state: TermItState) => state.configuration.language
-  );
+  const [label, setLabel] = useState({} as MultilingualString);
```

Comment:
I prefer generics (`useState<MultilingualString>({})`) over casting, but that's a minor detail.